.github/workflows/ci.yml

```
@@ -0,0 +1,168 @@
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r config/requirements.txt
+    
+    - name: Check Python syntax
+      env:
+        PYTHONPATH: src
+      run: |
+        python -m py_compile src/madspark/core/coordinator.py
+        python -m py_compile src/madspark/agents/*.py
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)